code/datums/brain_damage/paranoia.dm

```diff
@@ -0,0 +1,110 @@
+	if(!bad_touch)
+		had_bad_touch = TRUE
+	..()
+	//antag stuff//
+	antagonist.forge_objectives(owner.mind)
+	antagonist.greet()
+
+/datum/brain_trauma/special/renegade/on_life(seconds_per_tick, times_fired)
+	armed_check()
+	armored_check()
+
+	if(locate(/mob/living/carbon) in viewers(1, owner)-owner)
+		being_viewed = TRUE
+		too_close()
+		return
+
+	being_viewed = FALSE
+	time_close = 0
+	REMOVE_TRAIT(owner, TRAIT_NERVOUS_EYES, TRAUMA_TRAIT)
+
+/datum/brain_trauma/special/renegade/proc/too_close()
+	time_close += 30
+	if(time_close > 1800) //3 minutes
+		owner.add_mood_event("paranoid close", /datum/mood_event/paranoid_closesevere)
+		if(prob(10))
+			if(prob(50))
+				to_chat(owner, span_userdanger("[pick("They are standing near me for so long! They are totally plotting something!", "Oh my god! They are just standing there menacingly!", "Will they murder me? They are standing near me for so long!")]"))
+			else
+				owner.do_jitter_animation(30)
+				owner.playsound_local(get_turf(owner), 'sound/effects/singlebeat.ogg', 70, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
+				to_chat(owner, span_userdanger("You can feel your heart beating from paranoia!"))
+	else
+		owner.add_mood_event("paranoid close", /datum/mood_event/paranoid_close)
+		if(prob(10))
+			to_chat(owner, span_warning("[pick("Why are they so close to me? I'm nervous about this...", "They are plotting something? They are so close...", "I can't control my eyes, they're nervously looking around from this paranoia!")]"))
+			ADD_TRAIT(owner, TRAIT_NERVOUS_EYES, TRAUMA_TRAIT)
+
+/datum/brain_trauma/special/renegade/on_lose()
+	..()
+	owner.playsound_local(get_turf(owner), 'sound/ambience/ambiholy2.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
+	owner.clear_mood_event("paranoid close")
+	owner.clear_mood_event("paranoid armed")
+	owner.clear_mood_event("paranoid armored")
+	REMOVE_TRAIT(owner, TRAIT_NERVOUS_EYES, TRAUMA_TRAIT)
+	if(!had_bad_touch) //So we won't accidentaly delete the quirk they selected themselves, and not the one got from the paranoia
+		owner.remove_quirk(/datum/quirk/bad_touch)
+	owner.mind.remove_antag_datum(/datum/antagonist/renegade)
+
+/datum/brain_trauma/special/renegade/proc/armed_check()
+	var/list/find_weapons = owner.get_contents()
+	var/static/list/weapons = typecacheof(list(
+		/obj/item/gun,
+		/obj/item/melee,
+		/obj/item/grenade/frag,
+		/obj/item/circular_saw,
+		/obj/item/scalpel/advanced,
+		/obj/item/surgicaldrill,
+		/obj/item/storage/toolbox,
+		/obj/item/fireaxe,
+		/obj/item/spear,
+		/obj/item/knife,
+		/obj/item/switchblade,
+		/obj/item/flamethrower,
+	))
+
+	for(var/obj/item in find_weapons)
+		if(weapons[item.type])
+			total_time_armed += 30
+			owner.add_mood_event("paranoid armed", /datum/mood_event/paranoid_armed)
+			return
+	owner.add_mood_event("paranoid armed", /datum/mood_event/paranoid_notarmed)
+	if(prob(5))
+		to_chat(owner, span_warning("I better get some kind of weapon on my hand..."))
```

Comment:
Don't use first person perspective while casting to chat. It's only used in moodlets.
"You desperately need to get your hands on a weapon" etc.